tests/rmq/test_communicator.py

```diff
@@ -39,7 +39,6 @@ def loop_communicator():
         message_exchange=message_exchange,
         task_exchange=task_exchange,
         task_queue=task_queue,
-        decoder=functools.partial(yaml.load, Loader=yaml.Loader),
```

Comment:
> thanks! One thing is not clear to me: if the default loader in the plugin is the safe one, why is there no error before this PR. The safe loader will raise an error if the uuid is not serializable.

@superstar54 This line matters. If I didn't add the uuid_representer and uuid_constructor, after remove this unsafeloader decoder. The test hang because it can not deserialize the message.

Reply:
@superstar54 I guess your previous comment was gonna to reply this one?

I updated, by passing explicitly the decoder and encoder. Here actually it is a test, not the real code path used.

Reply:
It's still not clear to me. I am guessing you are adding the `uuid_representer` and `uuid_constructor` for safe load. But where is the test for them?

Reply:
> uuid_representer and uuid_constructor 

If I didn't add these and I remove the `decoder` line in loop_communicator, the test will fail (hang actually, because RMQ never get the correct message).

Reply:
Since you are adding a constructor, and your goal is to have the safe load, as suggested in my comment, it would be good to use `yaml.SafeLoader` instead of `yaml.FullLoader`, unless there is still another `message` type that also needs a constructor.

If there is another `message` type, you can add a constructor for it, as you already did for the `uuid`. It would be good if you could make a list of the possible message types.

If you think `yaml.FullLoader` is already safe enough, I am also fine.

Reply:
> If you think yaml.FullLoader is already safe enough, I am also fine.

The scope if this PR is to revert the original unnecessary change and using the default loader of kiwipy. 
The plan for the message passing part is to use msgpack to replace. So for this one I'll not changing the test above.